src/Vehicle/VehicleSetup/JoystickConfigCalibration.qml

```diff
@@ -13,11 +13,9 @@ import QtQuick.Dialogs
 import QtQuick.Layouts
 
 import QGroundControl
-
+import QGroundControl.Palette
```

Comment:
I don't think this module exists anymore, it's been merged with QGroundControl.Controls I believe